client/js/libs/jquery/stickyscroll.js

```diff
@@ -16,7 +24,8 @@
 					self.scrollBottom();
 				}
 			})
-			.scrollBottom();
+			.scrollBottom()
+			.data("window-sticky-fn", windowResizeHandler);
```

Comment:
Can't because `.sticky` might get called multiple times, so we have to pass it somehow accross the two functions.